docs/components/StatusLabel/StatusLabel.stories.mdx

```diff
@@ -65,9 +65,13 @@ where the StatusLabel is on the right, use the `end` alignment.
 
 ## Related components
 
-- [InlineLabel](../?path=/docs/components-status-and-feedback-inlinelabel--docs)
-  should be replaced with StatusLabel where possible to provide a more
-  intentional communication of status to users
+[InlineLabel](../?path=/docs/components-status-and-feedback-inlinelabel--docs)
```

Comment:
It looks like the notes section in this file has a similar statement. Based on the PR description that should also be removed

https://github.com/GetJobber/atlantis/blob/add-better-inlinelabel-status-label-docs/docs/components/StatusLabel/StatusLabel.stories.mdx#notes

<img width="661" alt="image" src="https://github.com/user-attachments/assets/7116a9bf-6f29-401a-9f74-fa3b699a0e8c">

Reply:
[Done](https://github.com/GetJobber/atlantis/pull/2092/commits/deb36b386f23c8473b351c2896db471f73fb2dc1)